.editorconfig

```diff
@@ -15,10 +15,14 @@ indent_size = 8
 indent_style = tab
 indent_size = 4
 
-[*.{less,yml}]
```

Comment:
We are using the same editorconfig within all repos, maybe that should be fixed on the other repos as well.